docs(concepts): improve concept pages for elements products with links to relevant docs MTA-1453 (#74)
scaleway/docs-content: compute/instances/concepts.mdx

## Boot-on-block

The boot-on-block feature allows Instances to boot from attached [Block storage](#block-volumes) volumes instead of local volumes. This feature is currently only available for [Enterprise Instances](#enterprise-instance). You can use this feature when you [create an Enterprise Instance](/compute/instances/how-to/use-enterprise-instances/#how-to-create-an-enterprise-instance)

</Concept>

<Concept>

Cloud-init is a multi-distribution package that [provides boot time customization for cloud servers](/compute/instances/how-to/use-boot-modes/#how-to-use-cloud-init). It enables automatic configuration of cloud instances as it boots into the cloud, turning a generic Ubuntu image into a configured server in a few seconds.

</Concept>

<Concept>

## Development Instance

> [Development Instances](https://www.scaleway.com/en/virtual-instances/development/) (aka DEV1 Instances) are reliable and flexible instances tuned to host your websites, applications and development environments.

Reviewer: ```suggestion
[Development Instances](https://www.scaleway.com/en/virtual-instances/development/) (aka DEV1 Instances) are reliable and flexible Instances tuned to host your websites, applications, and development environments.
```

## Enterprise Instance

> [Scaleway Enterprise Instances](https://www.scaleway.com/en/virtual-instances/enterprise/) (aka ENT1 Instances) are our latest high-end instances based on the AMD EPYC™ 7003 Series Processor. These Instances support the boot-on-block feature which allows them to boot from attached Block Storage volumes. Enterprise Instances come without local storage and you can add one or several Block Storage volumes depending on your requirements. Check out our documention: [how to use Enterprise Instances](/compute/instances/how-to/use-enterprise-instances/).

Reviewer 2: ```suggestion
[Scaleway Enterprise Instances](https://www.scaleway.com/en/virtual-instances/enterprise/) (aka ENT1 Instances) are our latest high-end instances based on the AMD EPYC™ 7003 Series Processor. These Instances support the boot-on-block feature which allows them to boot from attached Block Storage volumes. Enterprise Instances come without local storage and you can add one or several Block Storage volumes depending on your requirements. Check out our documentation: [how to use Enterprise Instances](/compute/instances/how-to/use-enterprise-instances/).
```

</Concept>

<Concept>

## Flexible IP

> Flexible IP addresses are public IP addresses which you can hold independently of any instance. When you create a Scaleway instance, its public IP address is also a flexible IP address. Flexible IP addresses can be attached to and detached from any of your instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users. Find out more: [how to use flexible IP addresses](/compute/instances/how-to/use-flexips/)

Reviewer: ```suggestion
Flexible IP addresses are public IP addresses which you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. Flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users. Find out more: [how to use flexible IP addresses](/compute/instances/how-to/use-flexips/)
```

## Placement Groups

> Placement Groups allow you to run multiple Compute instances each on a different physical hypervisor. Placement groups have two operating modes. The first one is called `max_availability`. It ensures that all the Compute instances that belong to the same cluster will not run on the same underlying hardware. The second one is called `low_latency` and does the exact opposite: it brings compute instances closer together to achieve higher network throughput. Find out more with our documentation on [how to use Placement Groups](/compute/instances/how-to/use-placement-groups/)

Reviewer: ```suggestion
Placement Groups allow you to run multiple Compute Instances each on a different physical hypervisor. Placement groups have two operating modes. The first one is called `max_availability`. It ensures that all the Compute Instances that belong to the same cluster will not run on the same underlying hardware. The second one is called `low_latency` and does the exact opposite: it brings Compute Instances closer together to achieve higher network throughput. Find out more with our documentation on [how to use Placement Groups](/compute/instances/how-to/use-placement-groups/)
```

</Concept>

<Concept> 

## Power-off Mode

> The Power-off mode [shuts down an Instance](/compute/instances/how-to/power-off-instance/) by transferring all data on the local volume of the instance to a volume store. The physical node is released back to the pool of available machines. The reserved flexible IP of the Instance remains available in the account.

Reviewer: ```suggestion
The Power-off mode [shuts down an Instance](/compute/instances/how-to/power-off-instance/) by transferring all data on the local volume of the Instance to a volume store. The physical node is released back to the pool of available machines. The reserved flexible IP of the Instance remains available in the account.
```

## Rescue Mode

Rescue mode restarts your server via the network on a minimal operating system. You can [use rescue mode](/compute/instances/how-to/use-boot-modes/#how-to-use-rescue-mode) to debug your server and recover your system data. Rescue mode creates a ramdisk with the content of a downloaded rootfs. You will have access to all your disks and will be able to perform debug and rescue actions. After disabling rescue mode, you will have to reboot your server. Your server needs to be running to switch to rescue mode.

</Concept>

<Concept>

## Reverse DNS

Reverse DNS is the opposite of classic "forward" DNS, and maps an IP address to a hostname. This can be useful if, for example, you want to send emails from your Instance. Find out how to [configure reverse DNS on your Instance](/compute/instances/how-to/configure-reverse-dns/) with our how-to documentation.

</Concept>

<Concept>

## Security Group

> Security groups allow you to [create rules to drop or allow traffic coming to and from your Instances](/compute/instances/how-to/use-security-groups/). You can set default policies for inbound and outbound traffic, and/or define rules to deal with traffic differently depending on its source. If you wish, you can then create your own private infrastructure by disabling the public IPs of your instances. Security groups can be [stateful](#stateful-security-groups) to concept or [stateless](#stateless-security-groups). Note that security groups for Bare Metal servers cannot be stateful.

Reviewer: ```suggestion
Security groups allow you to [create rules to drop or allow traffic coming to and from your Instances](/compute/instances/how-to/use-security-groups/). You can set default policies for inbound and outbound traffic, and/or define rules to deal with traffic differently depending on its source. If you wish, you can then create your own private infrastructure by disabling the public IPs of your Instances. Security groups can be [stateful](#stateful-security-groups) to concept or [stateless](#stateless-security-groups). Note that security groups for Bare Metal servers cannot be stateful.
```

</Concept>

<Concept>

## Snapshot

> A snapshot takes a picture of a specific volume at one point in time. For example you may have a server with one volume containing the OS and another containing the application data, and want to use different snapshot strategies on both volumes. [Creating snapshots of your volumes](/compute/instances/how-to/create-a-snapshot/) gives you total freedom of which volumes you want to back up, while [images](#image) are more convenient for full backups of your instance.

Reviewer: ```suggestion
A snapshot takes a picture of a specific volume at one point in time. For example you may have a server with one volume containing the OS and another containing the application data, and want to use different snapshot strategies on both volumes. [Creating snapshots of your volumes](/compute/instances/how-to/create-a-snapshot/) gives you total freedom of which volumes you want to back up, while [images](#image) are more convenient for full backups of your Instance.
```

</Concept>

<Concept>

## Stand-by Mode

> Stand-by mode is designed to temporarily stop an instance. When you [put a compute instance into Stand-by mode](/compute/instances/how-to/use-standby-mode/), the server is halted, but the compute instance remains allocated to your account and all data remains on the local storage of the instance.

Reviewer: ```suggestion
Stand-by mode is designed to temporarily stop an Instance. When you [put a compute instance into Stand-by mode](/compute/instances/how-to/use-standby-mode/), the server is halted, but the Compute Instance remains allocated to your account and all data remains on the local storage of the Instance.
```

## Stateful Security Groups

> Stateful Security Groups disregard the default policy and inbound/outbound rules if a connection is initiated from your instance. Traffic will always be permitted on connections that you initiated. This is useful for example if you want to initiate connections on many different and various ports for messaging, video streaming or other purposes. In this case, the traffic from/to that connection will not be blocked, even if you have defined a rule that would otherwise do so. Find out more with our dedicated documention: [how to use Security Groups](/compute/instances/how-to/use-security-groups/)

Reviewer: ```suggestion
Stateful Security Groups disregard the default policy and inbound/outbound rules if a connection is initiated from your Instance. Traffic will always be permitted on connections that you initiated. This is useful for example if you want to initiate connections on many different and various ports for messaging, video streaming, or other purposes. In this case, the traffic from/to that connection will not be blocked, even if you have defined a rule that would otherwise do so. Find out more with our dedicated documentation: [how to use Security Groups](/compute/instances/how-to/use-security-groups/)
```

 
</Concept>

<Concept>

## Stateless Security Groups

> Stateless Security Groups srictly apply the default policy and inbound/outbound rules, regardless of whether a connection is initiated from your instance or not. This is useful if you know exactly which ports you will always require for your instance, eg. port 22 for SSH. You can make your security group stateless, define a rule to allow traffic on port 22, and block incoming traffic on other ports regardless of whether a connection is initiated from your instance on this port or not. Find out more with our dedicated documention: [how to use Security Groups](/compute/instances/how-to/use-security-groups/)

Reviewer: ```suggestion
Stateless Security Groups strictly apply the default policy and inbound/outbound rules, regardless of whether a connection is initiated from your instance or not. This is useful if you know exactly which ports you will always require for your instance, eg. port `22` for SSH. You can make your security group stateless, define a rule to allow traffic on port `22`, and block incoming traffic on other ports regardless of whether a connection is initiated from your Instance on this port or not. Find out more with our dedicated documentation: [how to use Security Groups](/compute/instances/how-to/use-security-groups/)
```

## Volumes

> Volumes are the storage space of your instances. Each Virtual Cloud Instance requires a defined amount of local storage. Scaleway Instances benefit from local NVMe SSD volumes.

Reviewer: ```suggestion
Volumes are the storage space of your Instances. Each Virtual Cloud Instance requires a defined amount of local storage. Scaleway Instances benefit from local NVMe SSD volumes.
```
